taqueria-plugin-taquito/common.ts

```diff
@@ -0,0 +1,200 @@
+		);
+	}
+
+	let accountKey: string;
+	if (sender && sender !== 'default') {
+		const accounts = getSandboxInstantiatedAccounts(sandbox);
+		if (accounts.hasOwnProperty(sender)) {
+			accountKey = accounts[sender].secretKey;
+		} else {
+			return sendAsyncErr(
+				`${sender} is not an account instantiated in the current environment. Check .taq/config.json`,
+			);
+		}
+	} else {
+		let defaultAccount = getDefaultAccount(parsedArgs)(sandboxName);
+		if (!defaultAccount) {
+			const first = getFirstAccountAlias(sandboxName, parsedArgs);
+			if (first) {
+				defaultAccount = getSandboxAccountConfig(parsedArgs)(sandboxName)(first);
+				sendErr(
+					`Warning: A default account has not been specified for sandbox ${sandboxName}. Taqueria will use the account ${first} for this operation.\nA default account can be specified in .taq/config.json at JSON path: sandbox.${sandboxName}.accounts.default\n`,
+				);
+			}
+		}
+		if (!defaultAccount) {
+			return sendAsyncErr(`No accounts are available for the sandbox called ${sandboxName} to perform the operation.`);
+		}
+		accountKey = defaultAccount.secretKey;
+	}
+
+	const tezos = new TezosToolkit(sandbox.rpcUrl as string);
+	tezos.setProvider({
+		signer: new InMemorySigner(accountKey.replace(/^unencrypted:/, '')),
+	});
+	return tezos;
+};
+
+export const configureToolKitWithNetwork = async (
+	parsedArgs: UnionOpts,
+	networkName: string,
+	sender?: string,
+): Promise<TezosToolkit> => {
+	const network = getNetworkConfig(parsedArgs)(networkName);
+	if (!network) {
+		return sendAsyncErr(
+			`The current environment is configured to use a network called '${networkName}'; however, no network of this name has been configured in .taq/config.json.`,
+		);
+	}
+
+	let account: string;
+	if (sender && sender !== TAQ_ROOT_ACCOUNT) {
+		const accounts = getNetworkInstantiatedAccounts(network);
+		if (accounts.hasOwnProperty(sender)) {
+			account = sender;
+		} else {
+			return sendAsyncErr(
+				`${sender} is not an account instantiated in the current environment. Check .taq/config.json`,
+			);
+		}
+	} else {
+		account = TAQ_ROOT_ACCOUNT;
+	}
+
+	const tezos = new TezosToolkit(network.rpcUrl as string);
+	const key = await getAccountPrivateKey(parsedArgs, network, account);
+	await importKey(tezos, key);
+	return tezos;
+};
+
+export const getDeclaredAccounts = (parsedArgs: UnionOpts): Record<string, number> =>
+	Object.entries(parsedArgs.config.accounts).reduce(
+		(acc, declaredAccount) => {
+			const name: string = declaredAccount[0];
+			const tez: string | number = declaredAccount[1];
+			return {
+				...acc,
+				[name]: typeof tez === 'string' ? parseFloat(tez) : tez,
+			};
+		},
+		{} as Record<string, number>,
+	);
+
+export const getSandboxInstantiatedAccounts = (sandbox: SandboxConfig.t): Record<string, SandboxAccountConfig.t> =>
```

Comment:
I think we're going to stick with `account instances` over `instantiated accounts`.

Reply:
What do you mean?